duckduckgo/Android
# Feature/long press menu enhancements

Task/Issue URL: https://app.asana.com/0/608920331025315/800270854347855
Tech Design URL: 
CC: 

**Description**:
Small improvements to the browser's long press menu.

1. Show url that was pressed in the dialog title
1. Add option to copy url to clipboard
1. If long pressing on an anchor which is also an image, then adding the various options for opening new tabs as well as the option to download image.

**Steps to test this PR**:
1. Navigate to a site with links (e.g., https://bbc.co.uk/news)
1. Long press on a link
1. Verify url is shown in the title (it'll truncate if too long so test that too if you can)
1. Verify there is an option to copy url
1. Find a link which is an image type (e.g., https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_link_image). Long press and verify you have the options to
    1. Download image
    1. Copy link address
    1. Open in new tab
    1. Open in BG tab
    1. Share

<img width="745" alt="screen shot 2018-10-02 at 15 09 25" src="https://user-images.githubusercontent.com/1336281/46353924-3ba52a80-c655-11e8-9dfe-06a47ee30665.png">

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
